velox/common/io/CMakeLists.txt

```diff
@@ -11,4 +11,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-add_library(velox_common_io INTERFACE)
+find_package(Folly REQUIRED)
+include_directories(${FOLLY_INCLUDE_DIR})
```

Comment:
This is not required as well.

Reply:
Can you rebase with master? There seems to be a conflict `velox/common/io/CMakeLists.txt`